desktop/src/Desktop/Setup.hs

```diff
@@ -133,21 +142,28 @@ createNewWallet mMnemonic = Workflow $ do
       elClass "i" "fa fa-lg fa-refresh" blank
       text " Regenerate"
 
+  ePassword <- setPassword (fmap (fmap (\mn -> Crypto.sentenceToSeed mn Crypto.english "") . hush) mnemonic)
+    >>= holdDyn Nothing . fmap pure
+
+  let nextGate = liftA2 (||) (fmap isNothing dPassword) (fmap not stored)
```

Comment:
`gate beh ev` allows `ev` through when `beh` is `True`, so I thought `nextGate` would be `True` when allowing the "next" action. Maybe call it `disabledNext`